projects/plugins/jetpack/modules/videopress/class.videopress-player.php

```diff
@@ -339,15 +339,18 @@ private function html5_static() {
 			$html .= ' lang="' . esc_attr( $this->video->language ) . '"';
 		}
 		$html .= '>';
-		if ( ! isset( $this->options['freedom'] ) || $this->options['freedom'] === false ) {
+		if (
+			( ! isset( $this->options['freedom'] ) || $this->options['freedom'] === false )
+			&& isset( $this->video->videos, $this->video->videos->mp4 )
```

Comment:
This is redundant. `isset()` handles nested objects already.
```suggestion
			&& isset( $this->video->videos->mp4 )
```